src/amo/components/LandingPage.js

```diff
@@ -108,9 +115,22 @@ export class LandingPageBase extends React.Component {
   }
 }
 
-export function mapStateToProps(state, ownProps) {
+export function mapStateToProps(
+  state, ownProps, _apiAddonType = apiAddonType, _log = log
+) {
+  let addonType;
+  try {
+    addonType = _apiAddonType(ownProps.params.visibleAddonType);
+  } catch (err) {
+    if (err.message.match('not found in API_ADDON_TYPES_MAPPING')) {
```

Comment:
I'm not too worried about that because we use babel to compile all of our code to ES5, which has a lot of tweaks for browser safety. We use es6-error on web-ext and haven't run into any problems. Are Safari and Edge platforms we need to support?